lib/components.js

```diff
@@ -363,7 +363,9 @@ class DatePicker extends Component {
       'minimumDate',
       'minuteInterval',
       'mode',
-      'timeZoneOffsetInMinutes'
+      'timeZoneOffsetInMinutes',
+      'background',
```

Comment:
Got your point. Following that argument, shouldn't the ``DatePickerIOS`` locals live inside the config object too?